universe/Effect.cpp

```diff
@@ -3250,6 +3258,8 @@ void GenerateSitRepMessage::Execute(const ScriptingContext& context) const {
         }
     }
 
+    raw_sitrep_template = raw_sitrep_template && m_affiliation == AFFIL_SELF;
```

Comment:
Well, these are all intended to be personal advisories to the player who customized them, and if they could be sent to the other empires it might create confusion for the other player-- to them it would appear to be just another sitrep from the server.

I suppose I could instead just always override whatever was specified and make it to AFFIL_SELF no matter what, but making it an extra check is what had seemed best to me.